scipy/special/__init__.py

```diff
@@ -768,7 +768,7 @@
    exprel    -- Relative error exponential, (exp(x)-1)/x, for use when `x` is near zero.
    sinc      -- Return the sinc function.
 
-"""
+"""  # noqa: E501
```

Comment:
Probably want to add these exceptions to the pyproject.toml configuration (all __init__ files are weird).

Reply:
example: https://github.com/scikit-image/scikit-image/blob/main/pyproject.toml#L180